src/raftstore/store/peer.rs

```diff
@@ -191,6 +191,7 @@ pub struct Peer {
     raft_engine: Arc<DB>,
     cfg: Rc<Config>,
     peer_cache: RefCell<FlatMap<u64, metapb::Peer>>,
+    learner_cache: RefCell<FlatMap<u64, metapb::Peer>>,
```

Comment:
This is redundant. Use `peer_cache` instead.

Reply:
So should we rename `peer_cache` to `node_cache`? I think in our context, `peer` means it can vote.

Reply:
No, peer means `metapb::Peer`, which means a replica. Follower means it can vote.